code/modules/uplink/uplink_items.dm

```diff
@@ -1267,118 +1560,179 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
+			a Syndicate brand MMI, a straitjacket, and a muzzle."
+	item = /obj/item/storage/backpack/duffelbag/syndie/med/surgery
+	cost = 3
+
+/datum/uplink_item/tools/surgerybag_adv
+	name = "Advanced Syndicate Surgery Duffel Bag"
+	desc = "The Syndicate surgery duffel bag is a toolkit containing all surgery tools, surgical drapes, \
+			a Syndicate brand MMI, a straitjacket, a muzzle, and a full Syndicate Combat Medic Kit"
+	item = /obj/item/storage/backpack/duffelbag/syndie/med/surgery_adv
+	cost = 10
+
+/datum/uplink_item/tools/soap
+	name = "Syndicate Soap"
+	desc = "A sinister-looking surfactant used to clean blood stains to hide murders and prevent DNA analysis. \
+			You can also drop it underfoot to slip people."
+	item = /obj/item/soap/syndie
+	cost = 1
+	surplus = 50
+
+/datum/uplink_item/tools/syndie_glue
+	name = "Syndicate Glue"
+	desc = "A cheap bottle of one use syndicate brand super glue. \
+			Use on any item to make it undroppable. \
+			Be careful not to glue an item you're already holding!"
+	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
+	item = /obj/item/syndie_glue
 	cost = 4
 
-/datum/uplink_item/suits/hardsuit
-	name = "Syndicate Hardsuit"
-	desc = "The feared suit of a Syndicate nuclear agent. Features slightly better armoring and a built in jetpack \
-			that runs off standard atmospheric tanks. Toggling the suit in and out of \
-			combat mode will allow you all the mobility of a loose fitting uniform without sacrificing armoring. \
-			Additionally the suit is collapsible, making it small enough to fit within a backpack. \
-			Nanotrasen crew who spot these suits are known to panic."
-	item = /obj/item/clothing/suit/space/hardsuit/syndi
-	cost = 8
-	exclude_modes = list(/datum/game_mode/nuclear) //you can't buy it in nuke, because the elite hardsuit costs the same while being better
+/datum/uplink_item/tools/stimpack
+	name = "Stimpack"
+	desc = "Stimpacks, the tool of many great heroes, make you nearly immune to stuns and knockdowns for about \
+			5 minutes after injection."
+	item = /obj/item/reagent_containers/syringe/stimulants
+	cost = 5
+	surplus = 90
 
-/datum/uplink_item/suits/hardsuit/elite
-	name = "Elite Syndicate Hardsuit"
-	desc = "An upgraded, elite version of the Syndicate hardsuit. It features fireproofing, and also \
-			provides the user with superior armor and mobility compared to the standard Syndicate hardsuit."
-	item = /obj/item/clothing/suit/space/hardsuit/syndi/elite
-	cost = 8
-	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
-	exclude_modes = list()
+/datum/uplink_item/tools/phantomthief
+	name = "Syndicate Mask"
+	desc = "A cheap plastic mask fitted with an adrenaline autoinjector, which can be used by simply tensing your muscles"
+	item = /obj/item/clothing/glasses/phantomthief/syndicate
+	cost = 2
+
+/datum/uplink_item/tools/syndietome
```

Comment:
Nukies shouldn't have this

Reply:
Why not?

Reply:
Its free-hardsuit pening heals that should only be done with med-guns and the injector

Reply:
Let it be, it's a underpowered tool that doesn't work great for standard traitors anyway.
No real balance issue if one nukie is busy healing the company rather than shoot the enemy.